pkg/core/config/writer.go

```diff
@@ -249,11 +249,17 @@ type SplunkConfig struct {
 	// Splunk HTTP Event Collector token
 	Token string `yaml:"token"`
 	// Splunk source field value, description of the source of the event
-	Source string `yaml:"source"`
+	MetricsSource string `yaml:"source"`
 	// Splunk source type, optional name of a sourcetype field value
-	SourceType string `yaml:"sourceType"`
+	MetricsSourceType string `yaml:"sourceType"`
 	// Splunk index, optional name of the Splunk index to store the event in
-	Index string `yaml:"index"`
+	MetricsIndex string `yaml:"index"`
+	// Splunk index, specifically for traces (must be event type)
+	EventsIndex string `yaml:"eventsIndex"`
```

Comment:
This name and description is pretty confusing because in SignalFx _events_ are a thing distinct from traces.  If this is just saying which index traces go into can't it just be called something like `SpanIndex`?

Reply:
Splunk has two types of indexes: metrics and events. This relates more to the type of data stored in Splunk, and that needs to be disambiguated before data is ingested.

Events and traces will go to an event-type index. Metrics will go to a metrics index.